tests/test-searching.php

```diff
@@ -359,22 +359,19 @@ function test_query_paged_and_posts_per_page() {
 		);
 	}
 
-	/**
-	 * @ticket 18897
-	 */
 	function test_query_offset_and_paged() {
 		$this->assertEquals(
 			array(
-				'many-trackbacks',
-				'one-trackback',
-				'comment-test',
-				'lorem-ipsum',
-				'cat-c',
-				'cat-b',
-				'cat-a',
-				'cats-a-and-c',
-				'cats-b-and-c',
-				'cats-a-and-b',
+				'tag-c',
+				'tag-b',
+				'tag-a',
+				'tags-a-b-c',
+				'raw-html-code',
+				'simple-markup-test',
+				'embedded-video',
+				'contributor-post-approved',
+				'one-comment',
+				'no-comments',
```

Comment:
You need to revert the changes to this file. I'm sure there's a reason why you did this, so let's discuss in Slack, but this was very deliberate.